optimum/onnxruntime/__init__.py

```diff
@@ -146,7 +150,11 @@
         )
     else:
         from .modeling_diffusion import (
+            ORTDiffusionPipeline,
             ORTLatentConsistencyModelPipeline,
+            ORTPipelineForImage2Image,
```

Comment:
could you include it in `dummy_diffusers_objects.py` for cases when diffusers is not installed

Reply:
yes good catch